lib/intercom/api_operations/bulk/load_error_feed.rb

```diff
@@ -0,0 +1,13 @@
+module Intercom
+  module ApiOperations
+    module Bulk
+      module LoadErrorFeed
+        def load_error_feed(params)
```

Comment:
I think that `client.jobs.errors(id: ...)` might be slightly nicer api. Totally subjective though